### PR TITLE
End of Sprint 1 request move from Develop to Master

### PR DESCRIPTION
**Pull request by gervanpn**

### CODE REVIEW

File: big-game/src/app/horse-page/horse-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { NgbPanelChangeEvent } from '@ng-bootstrap/ng-bootstrap';
+//import { SignUpComponent } from '../sign-up/sign-up.component'
```
Comment (JuliaDunayeva) [inline big-game/src/app/horse-page/horse-page.component.ts:7]: extra space

Comment (JuliaDunayeva) [reply big-game/src/app/horse-page/horse-page.component.ts:7]: -3

File: big-game/src/app/horse-page/horse-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import { UserDataService } from '../services/user-data.service';
+
+@Component({
+  selector: 'app-horse-page',
+  templateUrl: './horse-page.component.html',
+  styleUrls: ['./horse-page.component.css']
+})
+
+export class HorsePageComponent implements OnInit {
+
```
Comment (JuliaDunayeva) [inline big-game/src/app/horse-page/horse-page.component.ts:26]: -1

File: big-game/src/app/horse-page/horse-page.component.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  
+  constructor(private router: Router, 
+    private http: HttpClient,
+    public colorService: ColorService, 
+    public breedService: BreedService,
+    public userDataService: UserDataService) { }
+
+  ngOnInit(): void {
+    //SignUpComponent.getColors();
+this.getBreeds();
```
Comment (JuliaDunayeva) [inline big-game/src/app/horse-page/horse-page.component.ts:44]: Formatting

Comment (JuliaDunayeva) [reply big-game/src/app/horse-page/horse-page.component.ts:44]: -3

File: big-game/src/app/horse-page/horse-page.component.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        console.log(result);
+        this.userData = result as Array<UserData>;
+        console.log(this.userData[0].password);
+      }
+    )
+    return this.userData;
+  }
+ 
+ public beforeChange($event: NgbPanelChangeEvent) {
+
```
Comment (JuliaDunayeva) [inline big-game/src/app/horse-page/horse-page.component.ts:126]: extra space -1

File: big-game/src/app/sign-up/sign-form/sign-form.component.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+.btn-group-lg > .btn,
+.btn-lg {
+  padding: 0.5rem 8rem;
+}
+
+.center {
+  text-align: center;
+  padding-right: 56%;
+}
+.form-check-input {
```
Comment (JuliaDunayeva) [inline big-game/src/app/sign-up/sign-form/sign-form.component.css:46]: space missing -2

File: big-game/src/app/sign-up/sign-form/sign-form.component.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+      <div formGroupName="userData">
+        <div class="form-group">
+          <input formControlName="username" placeholder="User Name" type="text" class="form-control" />
+          <span *ngIf="
+              !signForm.get('userData.username').valid &&
+              signForm.get('userData.username').touched
+            " class="text-danger">Please enter a valid username!
+          </span>
+        </div>
+
```
Comment (JuliaDunayeva) [inline big-game/src/app/sign-up/sign-form/sign-form.component.html:13]: inconsistent code -2

File: big-game/src/app/sign-up/sign-up.component.css [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+h5:first-child {
+    margin-top: 8px;
+    margin-bottom: 2px;
+}
+
+h5, .h5 {
+    font-size: 1.25rem;
+}
+
+/*****Moj*******/
```
Comment (JuliaDunayeva) [inline big-game/src/app/sign-up/sign-up.component.css:38]: We do not have Moj Element ) -1

File: big-game/src/app/sign-up/sign-up.component.ts [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+import { ColorService } from '../services/color.service';
+import { BreedService } from '../services/breed.service';
+import { Breed } from '../breed';
+
+@Component({
+	selector: 'app-sign-up',
+	templateUrl: './sign-up.component.html',
+	styleUrls: [ './sign-up.component.css' ]
+})
+export class SignUpComponent implements OnInit {
```
Comment (JuliaDunayeva) [inline big-game/src/app/sign-up/sign-up.component.ts:14]: No space -1

File: big-game/src/app/store-page/store-page.component.css [hunk 9]
```diff
@@ -0,0 +1 @@
+
```
Comment (JuliaDunayeva) [inline big-game/src/app/store-page/store-page.component.css:1]: ?
-1

File: big-game/src/app/sign-up/sign-form/sign-form.component.html (hunk 6 above)
Comment (gervanpn) [reply big-game/src/app/sign-up/sign-form/sign-form.component.html:13]: What part? - what does inconsistent mean? - how do we fix it/where? make this code consistent with what?

**Pull request merged**